core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -149,7 +149,9 @@ public static boolean allStrong(List<RexNode> operands) {
     return operands.stream().allMatch(Strong::isStrong);
   }
 
-  /** Returns whether an expression is definitely not true. */
+  /** Returns whether the analyzed expression will definitely not return true
+   * (equivalently, will definitely not return null or false) if
```

Comment:
The javadoc here was rather fine. The part that is broken is L82 `(equivalently, will definitely not return null or false)`. It should be `(equivalently, will definitely return null or false)`.

Reply:
Yes, sorry. Thanks for corrections.